### PR TITLE
bws-166: + added list of permission

### PR DESCRIPTION
**Pull request by bohdanshcherba**

### CODE REVIEW

**Review by what1s1ove — CHANGES_REQUESTED** [review]
you just need to add `Chip` component on permissions
why are there so many changes?

File: frontend/src/components/eam-configurate-worker/components/group-table/group-table.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
```
Comment (Yaroslaww-1) [inline frontend/src/components/eam-configurate-worker/components/group-table/group-table.tsx:1]: Please add a screenshot of groups table

**Review by Yaroslaww-1 — CHANGES_REQUESTED** [review]

**Comment by bohdanshcherba** [comment]
image

**Review by Yaroslaww-1 — DISMISSED** [review]

**Review by ania-leichenko — DISMISSED** [review]

**Review by what1s1ove — CHANGES_REQUESTED** [review]
in this PR we just need to add a `Chip` component to permissions table. without any other changes

File: frontend/src/components/eam-configurate-worker/components/group-table/group-table.tsx (hunk 1 above)
Comment (junojei) [reply frontend/src/components/eam-configurate-worker/components/group-table/group-table.tsx:1]:
```suggestion
import { FC } from 'react';
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]
in this PR we just need to add a `Chip` component to permissions table. without any other changes
we should have only 1-2 file changed. Now we have 17

**Pull request closed without merging**